packages/retarget-open-prs/index.js

```diff
@@ -0,0 +1,33 @@
+const { Octokit } = require( '@octokit/core' );
+
+async function retargetOpenPrs( repoOwner, repoName, from, to, accessToken ) {
+	const octokit = new Octokit( { auth: accessToken } );
+
+	const { data: pulls } = await octokit.request( 'GET /repos/{owner}/{repo}/pulls', {
+		owner: repoOwner,
+		repo: repoName,
+		base: from,
+	} );
+
+	for ( const pull of pulls ) {
+		const {
+			headers: { 'x-ratelimit-remaining': remainingRateLimit, 'x-ratelimit-reset': rateLimitReset },
+		} = await octokit.request( 'PATCH /repos/{owner}/{repo}/pulls/{pull_number}', {
+			owner: repoOwner,
+			repo: repoName,
+			pull_number: pull.number,
+			base: to,
+		} );
+
+		if ( remainingRateLimit === 0 ) {
+			const rateLimitResetDate = new Date( rateLimitReset * 1000 );
```

Comment:
The amount of time I'm observing locally can be up to an hour, which seems like an awful long time for a script to stall. Do you think we should determine a tolerance? Maybe if the time is within the next ~minute or so we wait but otherwise escape with the message?

Reply:
That's the thing, maybe I want the script to run in the background while I'm working on other things 😉 

We could have a config flag for that, and keep the default behavior as you described. What do you think?

Reply:
That sounds good to me!